### PR TITLE
Refactoring and fixed signup bug

### PR DESCRIPTION
**Pull request by ghost**
Fixed bug when creating female profile.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
     <application
-        android:name=".Be_you"
+        android:name=".ChatApplication"
```
Comment (kirantrathod) [inline app/src/main/AndroidManifest.xml:14]: Will it change the name of application?

File: app/src/main/res/drawable/ic_search.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#CCCCCC"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M15.5,14h-0.79l-0.28,-0.27C15.41,12.59 16,11.11 16,9.5 16,5.91 13.09,3 9.5,3S3,5.91 3,9.5 5.91,16 9.5,16c1.61,0 3.09,-0.59 4.23,-1.57l0.27,0.28v0.79l5,4.99L20.49,19l-4.99,-5zM9.5,14C7.01,14 5,11.99 5,9.5S7.01,5 9.5,5 14,7.01 14,9.5 11.99,14 9.5,14z"/>
+</vector>
```
Comment (kirantrathod) [inline app/src/main/res/drawable/ic_search.xml:5]: For what ic_serch.xml is?
Please provide comments or documentation!!

Comment (ghost) [reply app/src/main/res/drawable/ic_search.xml:5]: It let's you search user by name in the UsersActivity

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ghost) [reply app/src/main/AndroidManifest.xml:14]: It won't

File: app/src/main/res/drawable/ic_search.xml (hunk 2 above)
Comment (kirantrathod) [reply app/src/main/res/drawable/ic_search.xml:5]: Yeah!! it will great if we can search for user from that activity itself.

Comment (ghost) [reply app/src/main/res/drawable/ic_search.xml:5]: It's already done.

**Pull request merged**